src/app/modules/registration/components/numeric-input/numeric-input.component.ts

```diff
@@ -30,7 +30,7 @@ export class NumericInputComponent {
 
   get displayValue() {
     const converted = this.convert(this.value, 'from');
-    if (converted !== undefined) {
+    if (converted !== undefined && converted !== null) {
```

Comment:
Dette er så vidt jeg vet det samme som `converted != null`, som er lurt å bruke til sånne null-sjekker, da unngår man dette her. Men dette er også fint 👍

Reply:
Takk for tips! Dette må jeg lære meg. Har fikset koden nå.